form_alidation.js

```diff
@@ -0,0 +1,17 @@
+const emailErr = document.getElementById('emailErr');
+const submit = document.getElementById('submit');
+emailErr.style.color = 'red';
+
+function emailValidation() {
+  const email = document.getElementById('email').value;
+  if (email.toLowerCase() !== email) {
+    emailErr.innerHTML = 'Input the email in lowercase';
+  }
+  return false;
+}
+submit.addEventListener('click', (event) => {
+  if (!emailValidation()) {
+    event.preventDefault();
+  }
+});
+submit.addEventListener('click', emailValidation);
```

Comment:
- [ ] If the email is in upper case, I can see the error warning but if I enter the email in lower case nothing happens and the form is not being submitted. Please make sure to fix this issue.